Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'secure_headers', '~> 3.0.0'
 gem 'sidekiq'
 gem 'simple_form', github: 'amoose/simple_form', branch: 'feature/aria-invalid'
+gem 'sinatra', require: false
```

Comment:
I think this was inadvertent on your part. I just put it back in the test group and the deployment worked fine.